docs/book/v3/migration/v2-to-v3.md

```diff
@@ -105,3 +93,8 @@ function gravatarImage(
     string $rating = 'g'
 );
 ```
+
+#### Json
+
+The deprecated Json view helper has been removed.
+To encode data to Json for output in a view, you can call [`json_encode`](https://www.php.net/json_encode) directly.
```

Comment:
Should this mention of `toJson()` and `toArray()` when encoded data is an object?
https://github.com/laminas/laminas-json/blob/53ff787b20b77197f38680c737e8dfffa846b85b/src/Json.php#L99-L107

Reply:
Done!